interface/therapy_groups/therapy_groups_models/group_statuses_model.php

```diff
@@ -0,0 +1,58 @@
+<?php
```

Comment:
Where is namespace being used in that code, though? Note the path can be different from the namespace.